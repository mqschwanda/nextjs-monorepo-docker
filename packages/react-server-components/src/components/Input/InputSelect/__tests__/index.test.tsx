import { render } from '@testing-library/react';
import { InputSelect } from '@';

describe('@mqs/react-server-components', () => {
  describe('components', () => {
    describe('<InputSelect />', () => {
      it('renders', () => {
        const testId = InputSelect.name;

        const { getByTestId, asFragment } = render((
          <InputSelect
            className='test'
          >
            <option>
              { 'One' }
            </option>
            <option>
              { 'Two' }
            </option>
            <option>
              { 'Three' }
            </option>
          </InputSelect>
        ));

        const element = getByTestId(testId);
        expect(element).toBeTruthy();

        expect(asFragment()).toMatchSnapshot();
      });
    });
  });
});
