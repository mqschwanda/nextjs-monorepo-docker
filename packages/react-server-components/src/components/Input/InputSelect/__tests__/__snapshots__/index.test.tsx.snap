// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@mqs/react-server-components components <InputSelect /> renders 1`] = `
<DocumentFragment>
  <select
    class="select test"
    data-testid="InputSelect"
  >
    <option>
      One
    </option>
    <option>
      Two
    </option>
    <option>
      Three
    </option>
  </select>
</DocumentFragment>
`;
